src/mixins/raster-layer-poly-mixin.js

```diff
@@ -180,7 +186,9 @@ export default function rasterLayerPolyMixin(_layer) {
   }
 
   function usesAutoColors() {
-    return state.encoding.color.domain === "auto"
+    return state.encoding.color.domain === undefined
```

Comment:
I'm confused why this change is necessary? state.encoding.color.domain === "auto" will be false if state.encoding.color.domain is undefined, anyway.

Reply:
Good catch. Likely a holdover from when `state.encoding.color` could be undefined. I removed it.